src/commands/cmd_json.cc

```diff
@@ -189,6 +189,24 @@ class CommandJsonClear : public Commander {
   }
 };
 
+class CommandJsonToggle : public Commander {
+ public:
+  Status Execute(Server *svr, Connection *conn, std::string *output) override {
+    redis::Json json(svr->storage, conn->GetNamespace());
+
+    std::string path = (args_.size() > 2) ? args_[2] : "$";
+    auto s = json.Toggle(args_[1], path);
+
+    if (s.IsNotFound()) {
+      return Status::OK();
+    }
```

Comment:
Have you tested this case? I think here the connection will be blocked.

Reply:
Not yet. Writing more test cases for happy path. Will update asap with error scenarios.

Reply:
You can add `*output = redis::NilString()` before returning, just like in other commands.

Reply:
Sure will do